CMakeLists.txt

```diff
@@ -100,12 +121,14 @@ else()
   )
 endif()
 
-if("${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" VERSION_LESS "3.5")
-  message(
-    FATAL_ERROR
-      "Python v3 interpreter must be greater than or equal to 3.5. Found ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}"
-  )
-endif()
+# ~~~
+# if("${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" VERSION_LESS "3.5")
+#  message(
+#    FATAL_ERROR
+#      "Python v3 interpreter must be greater than or equal to 3.5. Found ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}"
+#  )
+# endif()
+# ~~~
```

Comment:
Why do we no longer need this?

Reply:
It was failing to find the python version in my local env for some reason. I just commented it out for now, I didn't want to spend the time investigating yet. Hopefully just due to me messing up my local env somehow.